cmd/podman/pod_create.go

```diff
@@ -11,6 +12,13 @@ import (
 	"github.com/urfave/cli"
 )
 
+var (
+	// CRI-O defaults to ipc, net and pid, but given pid is reliant on
+	// the first container running for the lifetime of the namespace,
+	// I am leaving it out of the default until pause container is implemented
+	DefaultKernelNamespaces = "ipc,net"
```

Comment:
I wouldn't have any of these - we should require an explicit --net=pod, --ipc=pod, etc